src/elements/content-sidebar/BoxAISidebarContent.tsx

```diff
@@ -93,6 +93,20 @@ function BoxAISidebarContent(props: ApiWrapperProps) {
             sendQuestion({ prompt: cacheQuestions[cacheQuestions.length - 1].prompt });
         }
 
+        if (recordAction) {
+            recordAction({
+                action: 'programmatic',
+                component: 'sidebar',
+                feature: 'answers',
+                target: 'loaded',
+                data: {
+                    items: items.map(item => {
+                        return { status: item.status, fileType: item.fileType };
+                    }),
```

Comment:
I remember from other PR that it was suggested we should filter out only supported files.